### PR TITLE
Jon Griffiths Homework 1 Complete 

### PR DESCRIPTION
**Pull request by jongriffiths**
CFSS Homework 1 
Jon Griffiths 
8 April 2019

### CODE REVIEW

**Comment by jongriffiths** [comment]
Note: The description of my process is included in the file "Jon Griffiths HW 1 Complete" above. I wasn't sure if you wanted it in there or in a comment here. As such, I've copied the text below in case that is more convenient:

Reporting my Process In completing this assignment, I initially found it difficult to see the changes in my local file reflected on the Github website. I found that this was occuring because my Github Desktop app required me to fill out a brief “summary” section prior to uploading the changes to the website. I did not notice this section at first so the the changes were not actually being sent to the Github website.

Since I am new to Github, many of the functions of the program seemed opaque. However, following this guide provided on canvas greatly simplified the process and made it far more easy to understand.

In terms of the exact process, I forked the Homework 1 folder from the Github account linked on course website into my own repository, then cloned that into my Github desktop app. Following that, I opened a new project in R Studio and associated the project with the directory just created by the Github app. I made changes to the README.md file using Markdown. The information on how to create bold letters, lists, links, and headers came from this helpful document. I found using Markdown to be the most intially intuitive part of the whole assignment. Once I wrote my information and did another pull from Github, I committed the changes and pushed them to Github app. From inside the app, I committed them to the master which made the edited document available online—only after discovering the aforementioned error with the “summary” section. Once my file was accessible from the website, I created a new pull request comparing the master to my forked assignment.

**Comment by cscrivner** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    x      |              |            |
| **Coding strategy**         |   x        |              |            |
| **Presentation**            |           |      x        |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |      x     |              |            |

Remarks:
Good job on HW1. All of your links worked well (including the clever link to Google). In the future, you can delete the "hw01 repository for homework 1" unless you just want to keep it. Cool that you've been able to visit so many places while campaigning!

File: README.html [hunk 1]
```diff
@@ -191,8 +191,12 @@
...
-<h1>Jon Griffiths</h1>
+<div id="homework-assignment-1" class="section level1">
+<h1>Homework Assignment 1</h1>
+<div id="jon-griffiths" class="section level2">
+<h2>Jon Griffiths</h2>
+<div id="computing-for-the-social-sciences" class="section level3">
+<h3>Computing for the Social Sciences</h3>
 <p><strong>Generic Info</strong> Hello everyone, my name is Jon Griffiths, and I’m a MAPSS student concentrating in American Politics. I mainly examine what factors increase or decrease voter registration and turnout rates. For my thesis, I’m analyzing the effects of the Automatic Voter Registration policies that have sprung up following the 2016 election in several states. If you wish to learn more about Automatic Voter Registration, check out this <a href="http://www.ncsl.org/research/elections-and-campaigns/automatic-voter-registration.aspx">website</a>. If you’re interested in learning about anything else, I strongly recommend this <a href="https://www.google.com/">website</a>.</p>
 <p>I’m from Cary, North Carolina but have spent the last 5 years Washington D.C. I’ve also done quite a bit of campaign work, so I’ve had extended stays in:</p>
```
Comment (sionbell) [inline README.html:204]: What campaign work have you done and in what capacities?

**Comment by sionbell** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |      x     |              |            |

Remarks:

* Good job! All required elements are there, including a good joke. 
* I knew you lived in DC for some time but didn't know that you also lived in various other places outside NC and DC. 
* Maybe include a picture next time!
* I don't know what was up with the "summary" requirement that confused you initially, but it seems to have worked out.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,25 @@
 # hw01
 Repository for homework 01
+
+#Homework Assignment 1
+## Jon Griffiths
+### Computing for the Social Sciences 
+### 8 April 2019
+
+**Generic Info**
+Hello everyone, my name is Jon Griffiths, and I'm a MAPSS student concentrating in American Politics. I mainly examine what factors increase or decrease voter registration and turnout rates. For my thesis, I'm analyzing the effects of the Automatic Voter Registration policies that have sprung up following the 2016 election in several states. If you wish to learn more about Automatic Voter Registration, check out this  [website](http://www.ncsl.org/research/elections-and-campaigns/automatic-voter-registration.aspx). If you're interested in learning about anything else, I strongly recommend this [website](https://www.google.com/).
```
Comment (braedenmc1) [inline README.md:10]: I appreciate the Google joke, very clever.

**Review by braedenmc1 — COMMENTED** [review]
I was unable to figure out how to paste the rubric, so here we go:

In terms of formatting, your HW1 was easy to follow and incorporated the 4 requirements listed in the evaluation on the course website. Your reflections also documented the Markdown commands, such as your website links. All throughout the assignment, your code was easy to follow.

Creativity: I liked the Google joke a bit too much, it was pretty good. You really established your project around your research and why it is personally important to you. It read like a story.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,25 @@
...
+
+I'm from Cary, North Carolina but have spent the last 5 years Washington D.C. I've also done quite a bit of campaign work, so I've had extended stays in:
```
Comment (pcuppernull) [inline README.md:12]: My parents live in Wilmington, NC! Love getting away from the winter weather to go visit them.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,25 @@
...
+* Des Moines, Iowa
+* Carroll, Iowa
+* Las Vegas, Nevada
+* Camden, New Jersey
+* Suffolk, Virginia
+
+**Reporting my Process**
+In completing this assignment, I initially found it difficult to see the changes in my local file reflected on the Github website. I found that this was occuring because my Github Desktop app required me to fill out a brief "summary" section prior to uploading the changes to the website. I did not notice this section at first so the the changes were not actually being sent to the Github website. 
+
+Since I am new to Github, many of the functions of the program seemed opaque. However, following this [guide provided on canvas](https://cfss.uchicago.edu/faq/homework-guidelines/#homework-workflow) greatly simplified the process and made it far more easy to understand.
```
Comment (pcuppernull) [inline README.md:23]: I agree - I am also new to Github and found the guide helpful.

**Review by pcuppernull — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    x      |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |           |       x       |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |    x      |              |            |

Remarks:

* Really nice job! Tough to find points for improvement
* I like how you formatted your header and will likely use that for my next assignment.
* I'd try including a picture next time as well!
* I don't have too much guidance to offer on figuring out Github, but hopefully it becomes easier moving forward!

**Pull request closed without merging**